postgresql/devpi_postgresql/main.py

```diff
@@ -11,6 +11,7 @@
 import pg8000
 import py
 import time
+from distutils.util import strtobool
```

Comment:
``devpi_server.model`` has ``ensure_boolean``, I think it would be more consistent to use it, as everything else in devpi uses it.

Reply:
OK, it now uses ensure_boolean.